### DISCUSSION TITLE
Error on installation

### DISCUSSION THREAD

**Discussion by Just-inc-ace** [gicamm/homeassistant-comelit#21] [General]
Hi,

I get the following error on installation:

Logger: custom_components.comelit.hub
Source: custom_components/comelit/hub.py:205 
Integration: Comelit Hub/Vedo (documentation) 
First occurred: 21:03:27 (1 occurrences) 
Last logged: 21:03:27

'sessiontoken'

What could be the reason?

**Comment by gicamm** [comment]
I'm sorry for the considerable delay.
Something wrong happened with notifications.

You have to configure the integration. You need to add the configuration in **configuration.yml** file.
So you need to pass a few parameters such as the hub username, password, serial, and address.

Did you look at the [wiki](https://github.com/gicamm/homeassistant-comelit/wiki)?
Please let me know if I can help you.
